as2_core/src/utils/tf_utils.cpp

```diff
@@ -105,7 +105,18 @@ geometry_msgs::msg::TransformStamped getTransformation(
 TfHandler::TfHandler(as2::Node * _node)
 : node_(_node)
 {
-  tf_buffer_ = std::make_shared<tf2_ros::Buffer>(_node->get_clock());
+  double tf_buffer_length = 10.0;
+  if (!_node->has_parameter("tf_buffer_length")) {
```

Comment:
i would suggest to try to use compound parameters name such as tf_hander.buffer_lenght or something like that to be more easy to know what is this parameter affecting